DisplaySettings/DisplaySettings.cpp

```diff
@@ -2670,7 +2670,7 @@ namespace WPEFramework {
 		string sMode = parameters["mode"].String();
                 string sLevel = parameters["level"].String();
                 dsVolumeLeveller_t volumeLeveller;
-                if ((Utils::isValidUnsignedInt ((char*)sMode.c_str()) == false) || (Utils::isValidUnsignedInt ((char*)sMode.c_str()) == false)) {
+                if (!Utils::isValidUnsignedInt(sMode.c_str())) {
                     LOGWARN("mode and level should be an unsigned integer");
```

Comment:
We probably should check sLevel as well.

Reply:
Done